shared_model/bindings/model_transaction_builder.hpp

```diff
@@ -61,6 +61,16 @@ namespace shared_model {
        */
       ModelTransactionBuilder quorum(interface::types::QuorumType quorum);
 
+      /**
+       * Sets batch meta
+       * @param type one of ATOMIC or ORDERED
+       * @param hashes vector of hashes of transactions in this batch
```

Comment:
You should either use dash "hashes - vector of hashes..." or reformulate the sentence, so that argument's name would be a correct part of it. The same for ```type``` argument slightly upwards